media-gfx/povray/povray-3.7.0.8.recipe

```diff
@@ -77,14 +76,24 @@ BUILD()
 {
 	cd unix
 	./prebuild.sh
+	autoupdate
 	cd ..
 
-	export CXXFLAGS="-std=c++11 -DBOOST_NO_CXX11_CONSTEXPR -D_BSD_SOURCE"
 	export LDFLAGS="-lbsd -lnetwork"
+	export CXXFLAGS="-std=c++11 -DBOOST_NO_CXX11_CONSTEXPR -D_BSD_SOURCE"
+	BoostLibLctr=$libDir
+	KeepEXR=""
+
+	if [ $effectiveTargetArchitecture = x86_gcc2 ]; then
+		BoostLibLctr=$BoostLibLctr/x86
+		KeepEXR="--without-openexr"
+	fi
 
 	runConfigure ./configure \
 		COMPILED_BY="HaikuPorts" \
-		--with-boost-libdir=`finddir B_SYSTEM_LIB_DIRECTORY`
+		--disable-optimiz \
```

Comment:
What's this?

Reply:
Disables its default -march=native setting.